src/Factory.php

```diff
@@ -2,24 +2,28 @@
 
 namespace Chromatic\OrangeDam;
 
+use Chromatic\OrangeDam\Endpoints\{AssetLink, DataTable, Endpoint, MediaFile, OAuth2, ObjectManagement, Search};
+use Chromatic\OrangeDam\Exceptions\{OrangeDamException, OrangeDamUnimplementedEndpointException};
 use Chromatic\OrangeDam\Http\Client;
-use Chromatic\OrangeDam\Endpoints\Endpoint;
-use Chromatic\OrangeDam\Exceptions\OrangeDamException;
 
 /**
  * Class Factory.
  *
- * @method \Chromatic\OrangeDam\Endpoints\Search       search()
- * @method \Chromatic\OrangeDam\Endpoints\OAuth2       oAuth2()
- */
+ * Manages the connection to Orange DAM API and a library of endpoints.
+*/
 class Factory
 {
+    /**
+     * Project URL
+     */
+    final public const PROJECT_URL = 'https://https://github.com/ChromaticHQ/orange-dam-php';
+
     /**
      * Client instance.
      *
-     * @var Client
+     * @var \Chromatic\OrangeDam\Http\Client
```

Comment:
Is this needed since we are now type-hinting the property?

Reply:
Not needed. I'm surprised rector didn't complain since it complained about other instances of this. 🤔